### ISSUE TITLE
[Bug] Invalid Attribute Type with resource citrix_application

### ISSUE THREAD

**Issue by cedfont** [citrix/terraform-provider-citrix#125]
with provider version 1.0.3 I get an error/warning with the resource “citrix application”

```
╷
│ Warning: Invalid Attribute Type
│ 
│   with citrix_application.published_application,
│   on main.tf line 1, in resource "citrix_application" "published_application":
│    1: resource "citrix_application" "published_application" {
│ 
│ Attribute type not supported: schema.Int32Attribute
╵
```

Module looks like this:
```hcl
#main.tf from module
resource "citrix_application" "published_application" {
  name                    = var.citrix_application_name
  description             = var.citrix_application_description
  published_name          = var.citrix_application_published_name
  application_folder_path = var.citrix_application_folder_path
  installed_app_properties = {
    command_line_arguments  = var.citrix_application_command_line_arguments
    command_line_executable = var.citrix_application_command_line_executable
    working_directory       = var.citrix_application_working_directory
  }
  delivery_groups = var.citrix_deliverygroup_name
  # icon            = comming soon
  limit_visibility_to_users = var.citrix_application_visibility
}
```

No error with Version 1.0.0

Is delivery_groups_priority regquired in 1.0.3?
I have only one delivery group in the variable string

**Comment by AlanCitrix** [comment]
Thanks for reporting this, we can reproduce the issue. We'll get a fix for it in the next release!

**Comment by yashbhokare-citrix** [comment]
Hi @cedfont,

This issue have been fixed in release v1.0.4. Here's the release note - https://github.com/citrix/terraform-provider-citrix/releases/tag/v1.0.4 and the version can be found here - https://registry.terraform.io/providers/citrix/citrix/1.0.4.

Closing this issue, feel free to open any issues or if you continue facing this issue.

Thanks,
Yash Bhokare
